app/controllers/dashboard_controller.rb

```diff
@@ -1,6 +1,9 @@
 class DashboardController < ApplicationController
 
   def index
+    if current_user.inactive?
```

Comment:
My name is Barry Allen. I recently was exposed to things that made me fast. Like, super fast. Faster than Sonic. Screw that hedgehog. I am the Flash.

![tumblr_nfpo8gisdy1qbi37ro1_250](https://user-images.githubusercontent.com/32661560/37940386-9848bdee-3125-11e8-82c1-f1e62944e8c1.gif)